satpy/etc/composites/virr.yaml

```diff
@@ -0,0 +1,43 @@
+sensor_name: visir/virr
+
+modifiers:
+  sunz_corrected:
+    compositor: !!python/name:satpy.composites.SunZenithCorrector
+    prerequisites:
+    - name: solar_zenith_angle
```

Comment:
Does `sunz_corrected` need to be defined? Isn't it defined and used from `visir.yaml`?

Reply:
`sunz_corrected` is defined differently in visir.yaml: visir.yaml uses "optional_prerequisites" and virr.yaml uses "prerequisites". I'm not sure if there's a better way to specify that without copying the few extra lines of code over. On top of that I don't know how virrs.yaml would use visir.yaml's `sunz_corrected`.

Reply:
Ah good point. This is fine by me then.